src/components/profile/ProfilePrivacy.js

```diff
@@ -1,21 +1,45 @@
 // @flow
-import React, { useEffect, useState } from 'react'
+
+// libraries
+import React, { useEffect, useMemo, useState } from 'react'
 import { RadioButton } from 'react-native-paper'
 import { TouchableOpacity } from 'react-native'
 import { startCase } from 'lodash'
+
+// custom components
+import Wrapper from '../common/layout/Wrapper'
+import { CustomButton, Icon, Section, Text } from '../common'
+import { BackButton } from '../appNavigation/stackNavigation'
+import BorderedBox from '../common/view/BorderedBox'
+
+// hooks
+import useOnPress from '../../lib/hooks/useOnPress'
+import { useDialog } from '../../lib/undux/utils/dialog'
+
+// utils
 import userStorage from '../../lib/gundb/UserStorage'
 import logger from '../../lib/logger/pino-logger'
-import { BackButton } from '../appNavigation/stackNavigation'
 import { withStyles } from '../../lib/styles'
-import { CustomButton, Icon, Section, Text } from '../common'
 import { fireEvent, PROFILE_PRIVACY } from '../../lib/analytics/analytics'
-import { useDialog } from '../../lib/undux/utils/dialog'
-import Wrapper from '../common/layout/Wrapper'
+import GDStore from '../../lib/undux/GDStore'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import { isSmallDevice } from '../../lib/utils/mobileSizeDetect'
+
+// assets
+import unknownProfile from '../../assets/unknownProfile.svg'
 import OptionsRow from './OptionsRow'
 
-const TITLE = 'PROFILE PRIVACY'
+// initialize child logger
 const log = logger.child({ from: 'ProfilePrivacy' })
 
+// get face record ID from userStorage and make it shorter if small device detected
+let faceRecordId = ''
+try {
+  // try-catch hack to pass tests
+  faceRecordId = userStorage.getFaceIdentifier()
```

Comment:
please do not use hacks.
jest provides function mocks. you could just mock userStorage.getFaceIdentifier in the tests calling it